packages/hash/frontend/src/pages/[account-slug]/types/new/entity-type.page.tsx

```diff
@@ -71,9 +72,11 @@ const Page: NextPageWithLayout = () => {
     shouldFocusError: true,
     mode: "onSubmit",
     reValidateMode: "onSubmit",
+    defaultValues: {
+      name: router.query.name as string,
```

Comment:
`name` is not definitely defined here

Reply:
Addressed here https://github.com/hashintel/hash/pull/1492/commits/d83d521d7167144f024f073e894990567dfd4ed0